### PR TITLE
[release] 20231004

### PR DESCRIPTION
**Pull request by jiqiang90**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ ] I have tested locally
- [ ] I have performed a self review of my changes
- [ ] Updated any relevant documentation
- [ ] Linked to any relevant issues
- [ ] I have added tests relevant to my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] My code is up to date with the base branch
- [ ] I have updated relevant changelogs. [We suggest using chan](https://github.com/geut/chan/tree/main/packages/chan)

### CODE REVIEW

File: packages/types/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.0] - 2023-10-04
```
Comment (stwiname) [inline packages/types/CHANGELOG.md:9]: I think this should be a patch release

File: packages/node/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.2] - 2023-10-04
+### Fixed
+- Fix reindex service without poi feature with `node-core`
+
+### Changed
+- Version bump with `types-core` 0.1.1
```
Comment (stwiname) [inline packages/node/CHANGELOG.md:14]: I don't think this applies here

File: packages/common-ethereum/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.0] - 2023-10-04
+### Changed
+- Version bump with `types-core` 0.1.1
```
Comment (stwiname) [inline packages/common-ethereum/CHANGELOG.md:11]: I don't think this applies here. Its common thats updated

File: packages/common-ethereum/CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.0] - 2023-10-04
```
Comment (stwiname) [inline packages/common-ethereum/CHANGELOG.md:9]: I think this should be a patch release

File: packages/node/CHANGELOG.md (hunk 2 above)
Comment (jiqiang90) [reply packages/node/CHANGELOG.md:14]: I think suppose be `types-ethereum` and `common-ethereum`

File: packages/common-ethereum/CHANGELOG.md (hunk 4 above)
Comment (stwiname) [inline packages/common-ethereum/CHANGELOG.md:9]:
```suggestion
## [3.0.1] - 2023-10-04
```

**Review by stwiname — APPROVED** [review]

**Pull request merged**